### DISCUSSION TITLE
fastcampus 강의중 누락된 CH013-02. [실습] auth0-redis-mysql PDF 파일 업로드 부탁드립니다

### DISCUSSION THREAD

**Discussion by seankim01** [yoonjeong-kwon/fastcampus-kubernetes#14] [Q&A] [answered]
fastcampus 통해 좋은 강의 잘 듣고 있습니다.
아래 PDF 파일이 누락된것 같아, Fastcampus 내에 '강의자료' 에 업로드 부탁 드릴 수 있을지요?
CH013-02. [실습] MSA 설계 및 앱 개발 하기(auth0-redis-mysql) 2시간 가까운 실습인데
일일이 타이핑 하기에는 너무 힘들듯 합니다, 시간되실때 다른 실습처럼, PDF 파일 자료실에 업로드 부탁드립니다
정말 좋은 강의 감사드립니다

**Comment by yoonjeong-kwon** [comment, marked as answer]
@seankim01 님, 안녕하세요.
제가 진행한 쿠버네티스 초격차 기본 과정은 CH05까지만 있고 redis나 mysql을 사용하는 실습은 존재하지 않습니다.
아무래도 다른 강좌에 대한 문의이신 것 같습니다. 😊 
다시 확인 부탁드려요. 좋은 하루되세요.
